src/changelog/2024/08/enterprise-license-update.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Enforcing Enterprise Restrictions
+description: "A stricter approach to expired licenses"
+
+date: 2024-07-24 13:00:00.0
+
+authors: ["ben-hardill"]
+tags:
+    - license
+    - enterprise
+---
+
+Starting with the v2.8.0 release we will be taking a slightly stricter 
+approach to Self Hosted Licenses. Previously when a license expired the
+platform would continue to run with the same capabilities as when the
+license was valid.
+
+If you are currently running with older version with an expired license
+please contact the Sales team to discuss renewing your license before 
+upgrading. You can check your current license on the 
+Admin Settings -> Overview page.
+
```

Comment:
Do changelogs allow for a "fold"? Do we need a `<!--more-->` here?

Reply:
Changelogs don't normally have a fold from the previous examples